CHANGELOG.mkd

```diff
@@ -3,8 +3,9 @@ CHANGELOG
 
 Unreleased
 ----------
-- (RK-383) Remove deprecated `basedir` method from Puppetfile DSL. Users should use `environment_name` instead. [#1254](https://github.com/puppetlabs/r10k/pull/1254)
 
+- (RK-383) Remove deprecated `basedir` method from Puppetfile DSL. Users should use `environment_name` instead. [#1254](https://github.com/puppetlabs/r10k/pull/1254)
+- (RK-386) Remove deprecated `bare` environment type. [#1235](https://github.com/puppetlabs/r10k/issues/1235)
```

Comment:
```suggestion
- (RK-386) Remove deprecated `bare` environment type. [#1253](https://github.com/puppetlabs/r10k/issues/1253)
```

Reply:
Good catch 😉

Reply:
oh I was wrong, the _issue_ _is_ #1235 lol

Reply:
weird coincidence 😅